core/src/mindustry/logic/LMarkerControl.java

```diff
@@ -0,0 +1,39 @@
+package mindustry.logic;
+
+public enum LMarkerControl{
+    create("type", "x", "y"),
+    remove,
+    hide,
+    show,
+    toggleVisibility,
+    setVisibility("true/false"),
+    setText("text"),
+    flushText,
+    setX("x"),
```

Comment:
I don't think any of these should have the set_ prefix, it's clear enough what they do anyway.